app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -175,7 +175,8 @@ def submitted_id_type
     end
 
     def id_type_requested
-      document_capture_session.passport_requested? ? 'passport' : 'state_id'
+      document_capture_session.passport_requested? ? Idp::Constants::DocumentTypes::PASSPORT :
+      'state_id'
```

Comment:
should we have `Idp::Constants:DocumentTypes::STATE_ID_CARD` here?

Reply:
After a quick trace through where this method is called and ends up, we can take two paths:
The quick path would be to make a constant for `state_id` no card.
The other path would be to investigate all the uses of `idType` in the react portion of document capture. Starting at `app/javascript/packages/document-capture/context/upload.tsx`